### PR TITLE
bpo-38872: Document exec symbol for codeop.compile_command

### PR DESCRIPTION
**Pull request by nanjekyejoannah**
Original author :  Андрей Беньковский, abenkovskii@gmail.com .

https://bugs.python.org/issue38872

### CODE REVIEW

**Review by nanjekyejoannah — COMMENTED** [review]
Requesting a look at this from @berkerpeksag

File: Doc/library/code.rst [hunk 1]
```diff
@@ -56,8 +56,8 @@ build applications which provide an interactive interpreter prompt.
 
    *source* is the source string; *filename* is the optional filename from which
    source was read, defaulting to ``'<input>'``; and *symbol* is the optional
-   grammar start symbol, which should be either ``'single'`` (the default) or
-   ``'eval'``.
+   grammar start symbol, which should be either ``'single'`` (the default), ``'eval'``
```
Comment (berkerpeksag) [inline Doc/library/code.rst]: I'd suggest dropping the "either" here.

File: Doc/library/codeop.rst [hunk 2]
```diff
@@ -43,8 +43,11 @@ To do just the former:
    :exc:`OverflowError` or :exc:`ValueError` if there is an invalid literal.
 
    The *symbol* argument determines whether *source* is compiled as a statement
-   (``'single'``, the default) or as an :term:`expression` (``'eval'``).  Any
-   other value will cause :exc:`ValueError` to  be raised.
+   (``'single'``, the default), as a sequence of statements (``'exec'``) or
+   as an :term:`expression` (``'eval'``).  Any other value will
+   cause :exc:`ValueError` to  be raised. If *symbol* is ``'single'`` and *source*
```
Comment (berkerpeksag) [inline Doc/library/codeop.rst]: The sentence starting with "If symbol is 'single' [...]" can also be dropped. I think these values are already documented at https://docs.python.org/3/library/functions.html#compile so we can keep these docs brief.

**Review by berkerpeksag — CHANGES_REQUESTED** [review]
This looks good to me, thanks! I just left two comments where we can drop some words just to make the docs more brief.

**Comment by berkerpeksag** [comment]
Also, do you think it's worth to backport this to 3.7 and 3.8?

**Comment by nanjekyejoannah** [comment]
> Also, do you think it's worth to backport this to 3.7 and 3.8?

yeah this SGTM.

**Review by berkerpeksag — APPROVED** [review]
LGTM, nice work, thanks!

**Pull request merged**